### PR TITLE
CV readme

### PR DESCRIPTION
**Pull request by AlanFFM**
Hi there! This is for one of the assignments. Cheers!

### CODE REVIEW

File: student-work/2023/Alan-Bariman/readme.md [hunk 1]
```diff
@@ -1 +1,86 @@
...
+PHONE:
++972-053-480-7177—Israel
++1-514-710-8088—Canada
+
+EMAIL:
+Alan.b.r@outlook.com (Work Address)
+Alan.bariman@mail.mcgill.ca<br />
+
+NOTABLE SKILLS
+•	Mastery of: Excel, Word, Photohosop, Illustrator, Latik, MasterCam, Fusion360, Lightroom, and Blender.<br />
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Alan-Bariman/readme.md:21]: All bullets are as follows

```* text ``` so it becomes
* text
* text

Comment (Laura-Novich-OBW) [reply student-work/2023/Alan-Bariman/readme.md:21]: one space between the bullet and the text

File: student-work/2023/Alan-Bariman/readme.md [hunk 2]
```diff
@@ -1 +1,86 @@
+ALAN BARIMAN
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Alan-Bariman/readme.md:1]: Consider using headings

```# Heading``` becomes
# Heading

**Comment by Laura-Novich-OBW** [comment]
Hi Alan, nice job. 
I strongly suggest you use the Markdown cheat sheets I provided in Google Classroom and make sure that your markdown is clean and highly organized. 

While you don't need to use massive amounts of HTML, you should use styles and formats which make the page appealing to a potential manager.

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
please address the comments

**Comment by AlanFFM** [comment]
Hi. returned from Mexico. Getting on this today.

**Review by Laura-Novich-OBW — APPROVED** [review]

**Pull request merged**